### DISCUSSION TITLE
swanctl --init with interactive eap id and credential

### DISCUSSION THREAD

**Discussion by psalty** [strongswan/strongswan#1583] [Q&A] [answered]
Hi. Is there any equivalent way to achieve 

`charon-cmd --host <hostname> --identity <id> [options]`

with swanctl?

Thanks

**Comment by tobiasbrunner** [comment, marked as answer]
What do you mean?

**Reply by psalty** [reply]
I mean when I do swanctl --init, is there any way to provide id and password from the command line so that I don't have to put in swanctl.conf

**Reply by tobiasbrunner** [reply]
Ah, no.
